README.md

```diff
@@ -24,54 +24,46 @@ module "runtime_connector" {
-| <a name="input_custom_external_id"></a> [custom\_external\_id](#input\_custom\_external\_id) | The external ID to use for AWS assume role validation. If unspecified, the connector generates an external ID and the Sym platform ensures it is unique. | `string` | `""` | no |
-| <a name="input_environment"></a> [environment](#input\_environment) | An environment qualifier for the resources this module creates, to support a Terraform SDLC. | `string` | n/a | yes |
-| <a name="input_policy_arns"></a> [policy\_arns](#input\_policy\_arns) | Map of logical identifiers to additional IAM Managed Policy ARNs to add to the runtime connector role. The identifiers are only used for managing Terraform state. | `map(string)` | `{}` | no |
-| <a name="input_sym_account_ids"></a> [sym\_account\_ids](#input\_sym\_account\_ids) | List of account ids that can assume the runtime role. By default, only Sym production accounts can assume the runtime role. | `list(string)` | <pre>[<br>  "803477428605"<br>]</pre> | no |
-| <a name="input_tags"></a> [tags](#input\_tags) | Additional tags to apply to resources | `map(string)` | `{}` | no |
+| <a name="input_environment_name"></a> [environment\_name](#input\_environment\_name) | The unique name of the environment in which you are deploying this Sym Runtime Role. (e.g. staging, or prod) | `string` | n/a | yes |
+| <a name="input_sym_account_id"></a> [sym\_account\_id](#input\_sym\_account\_id) | The AWS account ID that can assume the Sym Runtime Role. Defaults to the Sym Production AWS account ID. | `string` | `"803477428605"` | no |
+| <a name="input_tags"></a> [tags](#input\_tags) | Additional tags to apply to the AWS resources | `map(string)` | `{}` | no |
 
 ## Outputs
 
 | Name | Description |
 |------|-------------|
-| <a name="output_account_id"></a> [account\_id](#output\_account\_id) | The AWS account ID for this connector |
-| <a name="output_settings"></a> [settings](#output\_settings) | A map of settings to supply to a Sym Permission Context. |
+| <a name="output_sym_integration"></a> [sym\_integration](#output\_sym\_integration) | A [sym\_integration](https://registry.terraform.io/providers/symopsio/sym/latest/docs/resources/integration) resource that tells the Sym Runtime which AWS Role to assume to perform actions in your AWS account. For example, this can be used in sym\_runtime and sym\_secrets resources. |
+| <a name="output_sym_runtime"></a> [sym\_runtime](#output\_sym\_runtime) | A [sym\_runtime](https://registry.terraform.io/providers/symopsio/sym/latest/docs/resources/runtime) resource to be passed into your sym\_environment to enable the execution of AWS Strategies. |
+| <a name="output_sym_runtime_connector_role"></a> [sym\_runtime\_connector\_role](#output\_sym\_runtime\_connector\_role) | An [aws\_iam\_role](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_role) resource. This AWS IAM Role will be assumed by the Sym Runtime to execute operations in your AWS account. |
```

Comment:
kind of a nit by i think this should be named `role_arn`, the other outputs make sense to have a `sym_` prefix cause they are sym resources, i think `role_arn` is sufficient for this one.

Reply:
We're not outputting just the ARN, we're actually outputting the entire `aws_iam_role` resource, which is why it's named `sym_runtime_connector_role`